### PR TITLE
feature/SCAN-318-format-numbers-and-mobile-view-for-transfers-block

### PR DESCRIPTION
**Pull request by a-mikhailova**
remark: removed unused components

### CODE REVIEW

File: src/pages/Account/index.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { useApi } from '@app/hooks';
 import { normalizeSubstrate } from '@app/utils/normalizeAccount';
 
 import AccountDetailComponent from './components/AccountDetailComponent';
-import LastTransfersComponent from '../Main/components/LastTransfersComponent';
+import { LastTransfers } from '../Main/components/LastTransfers/LastTransfers';
```
Comment (a-mikhailova) [inline src/pages/Account/index.tsx:10]: change import to new component

File: src/components/Header.tsx [hunk 2]
```diff
@@ -17,6 +17,17 @@ const Header: FC = () => {
 
   const navigate = useNavigate();
 
+  const onLogoClick = useCallback(
```
Comment (a-mikhailova) [inline src/components/Header.tsx:20]: on the main page logo click the page must be reloaded

File: src/pages/Main/components/Collections/Collections.tsx [hunk 3]
```diff
@@ -72,6 +72,8 @@ export const Collections: VFC<CollectionsProps> = ({ searchString }) => {
     });
   }, [searchString, fetchMoreCollections, orderBy]);
 
+  if (!collections.length) return null;
```
Comment (a-mikhailova) [inline src/pages/Main/components/Collections/Collections.tsx:75]: we don't show entire block, if we have no data

File: src/pages/Main/components/TokensBlock/Tokens.tsx [hunk 4]
```diff
@@ -53,6 +53,8 @@ export const Tokens: VFC<TokensProps> = ({ collectionId, searchString }) => {
     pageSize: tokensLimit,
     searchString
   });
+  
+  if (!tokens?.length) return null;
```
Comment (a-mikhailova) [inline src/pages/Main/components/TokensBlock/Tokens.tsx:57]: we don't show entire block, if we have no data

**Review by kpozdnikin — APPROVED** [review]

**Pull request merged**